src/test/java/nextstep/subway/acceptance/StationAcceptanceTest.java

```diff
@@ -68,6 +56,18 @@ void createStation() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given
+        지하철역_생성_요청("기흥역");
+        지하철역_생성_요청("신갈역");
+
+        // when
+        ExtractableResponse<Response> response = 지하철역_목록조회_요청();
+
+        // then
+        List<String> stationNames = response.jsonPath().getList("name", String.class);
+        assertThat(stationNames).hasSize(2);
+        assertThat(stationNames).containsAnyOf("기흥역");
```

Comment:
해당 코드는 아래와 같이 해도 테스트를 통과합니다.
조금 더 명확하게 `containsExactly` 혹은 `containsExactlyInAnyOrder` 를 써보시는 건 어떨까요? 😄 

```suggestion
        assertThat(stationNames).containsAnyOf("기흥역", "역삼역");
```

Reply:
앗 이런 메서드도 있었군요 ㅎㅎ 감사합니다 😃